src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -102,17 +64,111 @@
  * To configure the check:
  * </p>
  * <pre>
- * &lt;module name=&quot;FallThrough&quot;/&gt;
+ * &lt;module name="FallThrough"/&gt;
  * </pre>
  * <p>
- * or
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() throws Exception {
+ *   int i = 0;
+ *   while (i &gt;= 0) {
+ *     switch (i) {
+ *       case 1:
+ *         i++;
+ *       case 2: // violation, previous case contains code but lacks
+ *               // break, return, throw or continue statement
+ *         i++;
+ *         break;
+ *       case 3: // OK
+ *         i++;
+ *         return;
+ *       case 4: // OK
+ *         i++;
+ *         throw new Exception();
+ *       case 5: // OK
+ *         i++;
+ *         continue;
+ *       case 6: // OK
+ *       case 7: // OK, previous case does not contain code
+ *         i++;
+ *         // OK, by default the last case might not have statement that transfer control
```

Comment:
```
 *       case 6: // OK case does not contain code
 *       case 7: // OK, by default the last case might not have statement that transfer control
 *         i++;
 *         
```

Reply:
Similar to what I just said in [this comment](https://github.com/checkstyle/checkstyle/pull/8847#discussion_r496970680):
```
    case 6:
        i++;
    case 7: // violation for case 6 AND case 7 occurs here!
        i++;
```

Reply:
Just use plural form
` // violations .....`

Reply:
Code fixed and images fixed according to [this](https://github.com/checkstyle/checkstyle/pull/8847#discussion_r497495087)